toolchain/check/testdata/impl/lookup/min_prelude/specialization_poison.carbon

```diff
@@ -23,6 +23,9 @@ impl forall [U:! type] U as I where .T = () {
   fn F[self: Self]() -> () { return (); }
 }
 
+// CHECK:STDERR: fail_final_poisoned_concrete_query_in_specific.carbon:[[@LINE+3]]:25: error: found `impl` that would change the result of an earlier use of `C*` as `I` [PoisonedImplLookupConcreteResult]
```

Comment:
Maybe add a TODO to diagnose as `` `C* as I` `` rather than `` `C*` as `I` ``.

Reply:
Ah ok I just made this work now with a `SpecificInterfaceIdAsRawType`